DOC: Remove unused ReadTheDocs (#650)
spacetelescope/hstcal: README.md

Calibration software for HST/WFC3, HST/ACS, and HST/STIS.

HSTCAL is a C-based package which is comprised of the science calibration
software in support of the Advanced Camera for Surveys (ACS), Space Telescope

> Imaging Spectrograph (STIS), and Wide Field Camera 3 pipelines. Initially, the

Reviewer: As for the other instruments, add (WFC3).

> Imaging Spectrograph (STIS), and Wide Field Camera 3 pipelines. Initially, the

Author: Lemme try this here and see if I can accept all your suggestions too in one go.
```suggestion
Imaging Spectrograph (STIS), and Wide Field Camera 3 (WFC3) pipelines. Initially, the
```

pipelines were written using C, but were encapsulated within the IRAF/STSDAS
environment, relying on IRAF to perform I/O and other basic interface
functions. HSTCAL replaces all the IRAF-based functionality with routines
based on the third-party package CFITSIO. This allows all the pipeline
software to be compiled and run without any dependence on IRAF. Not only
can HST data be processed using the C code directly via the C executables,
but the pipelines can also be run by using a high-level Python interface
(via [subprocess](https://docs.python.org/3/library/subprocess.html))

> that serves as thin wrappers for the C executables, which are availble

Author: ```suggestion
comprised of thin wrappers for the C executables. The wrappers are available
```

export jref="/grp/crds/cache/references/hst/"
```

> For more information, please see the [ACS Data Handbook](https://hst-docs.stsci.edu/acsdhb)

Reviewer: Suggestion: Just after the export jref line, you could add in this single location a reminder for the user that if they are running the software on a machine which is not connected to STScI the jref needs to be set to the location where the calibration reference files exist on their own machine.

Reviewer: Yes, strictly speaking, but I was hoping folks would get the general idea.  You can repeat or put the statement in a place where it is clear it applies to all the calibration reference files.

export oref="/grp/crds/cache/references/hst/"
```

For more information, please see the [STIS Data Handbook](https://hst-docs.stsci.edu/stisdhb)
and the [stistools documentation](https://stistools.readthedocs.io/).

## Other Software

> The following are also part of HST pipeline but not in this package:

Reviewer: Change: "following are also part of HST pipeline but not in this package" to "following are also part of the standard HST pipeline, but are independent software packages written in Python or Python layered on C: calcos, costools, and drizzlepac.

> The following are also part of HST pipeline but not in this package:

Author: ```suggestion
The following are also part of the standard HST pipeline, but are independent software
packages written in Python or Python layered on C:
```

> * COS pipeline: See [COS Data Handbook](https://www.stsci.edu/hst/cos/documents/handbooks/datahandbook/COS_cover.html) and [costools documentation](https://costools.readthedocs.io/)

Author: ```suggestion
* COS pipeline (`calcos` and `costools`): See [COS Data Handbook](https://www.stsci.edu/hst/cos/documents/handbooks/datahandbook/COS_cover.html) and [costools documentation](https://costools.readthedocs.io/)
```
